Makefile

```diff
@@ -1,5 +1,5 @@
 
-EXTVERSION = 1.8.2dev
+EXTVERSION = 1.8.4dev
```

Comment:
Is this supposed to be `1.8.3-dev`?

Reply:
I'm not sure, but this version number was not bumped for the previous release. I'll create a 1.8.4 release just to be on the safe side.